frontend/src/tests/components/eMIB/ActionViewEmail.test.js

```diff
@@ -61,29 +37,29 @@ describe("Email header", () => {
   const headerWithCc = <span>cc</span>;
 
   it("renders email's header with cc)", () => {
-    const wrapper = shallow(
-      <ActionViewEmail
-        responseType={EMAIL_TYPE.reply}
-        to={"to"}
-        cc={"cc"}
-        response={"response"}
-        reasonsForAction={"reasons"}
-      />
-    );
+    const wrapper = genWrapper(EMAIL_TYPE.reply, "cc");
 
     expect(wrapper.containsMatchingElement(headerWithCc)).toEqual(true);
   });
 
   it("renders email's header without cc)", () => {
-    const wrapper = shallow(
-      <ActionViewEmail
-        responseType={EMAIL_TYPE.reply}
-        to={"to"}
-        response={"response"}
-        reasonsForAction={"reasons"}
-      />
-    );
+    const wrapper = genWrapper(EMAIL_TYPE.reply, null);
 
     expect(wrapper.containsMatchingElement(headerWithCc)).toEqual(false);
   });
 });
+
+function genWrapper(responseType, cc) {
+  const actionStub = {
+    actionType: ACTION_TYPE.email,
+    reasonForAction: "reasons",
+    emailType: responseType,
+    emailTo: "to",
+    emailCc: cc,
+    emailBody: "reasons"
+  };
+
+  const wrapper = shallow(<ActionViewEmail action={actionStub} emailId={1} />);
```

Comment:
`return shallow(<ActionViewEmail action={actionStub} emailId={1} />);`

Reply:
fixed in e431ee1